### PR TITLE
Allow enabling computed mipmaps in `LargeTextureStore`

### PR DESCRIPTION
**Pull request by EVAST9919**
As [per discussion](https://discord.com/channels/188630481301012481/589331078574112768/1174532880408518838).

### CODE REVIEW

**Comment by bdach** [comment]
No testing whatsoever?

**Comment by EVAST9919** [comment]
Added 👌

File: osu.Framework.Tests/Visual/Sprites/TestSceneLargeTextureStore.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+namespace osu.Framework.Tests.Visual.Sprites
+{
+    public partial class TestSceneLargeTextureStore : FrameworkTestScene
+    {
+        private DependencyContainer dependencies = null!;
+
+        [BackgroundDependencyLoader]
+        private void load(IRenderer renderer, GameHost host, Game game)
+        {
+            dependencies.CacheAs(new LargeTextureStore(renderer, host.CreateTextureLoaderStore(new NamespacedResourceStore<byte[]>(game.Resources, "Textures")), manualMipmaps: false));
```
Comment (peppy) [inline osu.Framework.Tests/Visual/Sprites/TestSceneLargeTextureStore.cs:23]: Setting `manualMipamaps: true` doesn't change anything in the test.

I'd hope the test shows the difference between the two.

**Review by peppy — CHANGES_REQUESTED** [review]

File: osu.Framework.Tests/Visual/Sprites/TestSceneLargeTextureStore.cs (hunk 1 above)
Comment (EVAST9919) [reply osu.Framework.Tests/Visual/Sprites/TestSceneLargeTextureStore.cs:23]: >Setting manualMipamaps: true doesn't change anything in the test.

That's weird, for me the difference is very clear (I'm on D3D)

Comment (peppy) [reply osu.Framework.Tests/Visual/Sprites/TestSceneLargeTextureStore.cs:23]: May have been hot reload failing. But I'd also hope the test can be run in both modes. I don't want to have to change code each time I want to check a test.

Comment (EVAST9919) [reply osu.Framework.Tests/Visual/Sprites/TestSceneLargeTextureStore.cs:23]: Will do. Was just trying to ensure it works for you and not a renderer issue or something.

**Review by peppy — APPROVED** [review]

**Pull request merged**